nab-hibernate/src/test/java/ru/hh/nab/hibernate/HibernateTest.java

```diff
@@ -51,19 +55,19 @@ public void setName(String name) {
   public void test() throws PropertyVetoException {
     ComboPooledDataSource ds = new ComboPooledDataSource();
     ds.setJdbcUrl("jdbc:hsqldb:mem:" + getClass().getName());
-    ds.setDriverClass("org.hsqldb.jdbcDriver");
+    ds.setDriverClass(JDBCDriver.class.getName());
     ds.setUser("sa");
     ds.setPassword("");
 
-    Ejb3Configuration cfg = new Ejb3Configuration();
+    final Properties properties = new Properties();
```

Comment:
https://github.com/hhru/nuts-and-bolts/blob/master/example-app/src/test/java/ru/hh/nab/example/PersistenceTest.java